kura/org.eclipse.kura.core/OSGI-INF/system.xml

```diff
@@ -18,10 +18,12 @@
       <provide interface="org.eclipse.kura.system.SystemService"/>
    </service>
    <property name="service.pid" value="org.eclipse.kura.system.SystemService"/>
+<!--
```

Comment:
If it is not needed anymore, let's remove it

Reply:
done